gcloud-java-examples/src/main/java/com/google/cloud/examples/nio/CountBytes.java

```diff
@@ -0,0 +1,82 @@
+package com.google.cloud.examples.nio;
+
+import com.google.common.base.Stopwatch;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * CountBytes will read through the whole file given as input.
+ *
+ * <p>It's meant for testing that NIO correctly saturates your network link.
```

Comment:
I really meant that we can use this to see roughly how fast things go and to make sure we're not reading unnaturally slowly. I changed the text to reflect this more modest reality :)